ORST/Assets/Scripts/Core/Interactions/ForbiddenSpace/HandHeadProximity.cs

```diff
@@ -0,0 +1,91 @@
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace ORST.Core.Interactions
+{
+    public class HandHeadProximity : MonoBehaviour {
+        public event Action ForbiddenSpaceEntered;
+        public event Action ForbiddenSpaceExited;
+        public event Action<float> ForbiddenSpaceUpdated;
+
+        [SerializeField] private Collider m_InnerCollider;
+        [SerializeField] private Collider m_OuterCollider;
+
+        private readonly List<Transform> m_Intersector = new ();
```

Comment:
Extra space is not needed

```suggestion
        private readonly List<Transform> m_Intersector = new();
```